streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -883,6 +887,50 @@ public void statefulTopologyShouldCreateStateDirectory() throws Exception {
         startStreamsAndCheckDirExists(topology, true);
     }
 
+    @Test
+    public void shouldThrowIllegalArgumentExceptionOnEmptyTopology() {
+        assertThrows(
```

Comment:
IMHO, we should always get the exception and assert on the error message (otherwise, it exception might be throw because of. different reason the the test does not test what it is support to test -- we have seen this issue in the past).

Reply:
Ack